config/processors/syslog_log_security_sdwan.app.conf

```diff
@@ -5,78 +5,90 @@ input {
     address => VAR_PIPELINE_NAME
   }
 }
+
 filter {
   mutate {
     add_field => { "[event][module]" => "versa" }
     add_field => { "[event][dataset]" => "versa.sdwan_application" }
   }
-  # 1. start by copying the original message to [log][original].  we'll then pick the common fields and placing them under [tmp]. [tmp] is a temporary slot to work out the parsing
+  # 1. initial parsing
 
+  # a) start by copying the original message to log.original.  
+  # this needs to be done in two separate mutate steps in logstash > 8
+  # also, on logstash 8 / ecs 8 , as soon as you touch the message the original event is copied to event.original, so log.original and event.original are redundant. 
+  # both are kept for compatibility. this should be addressed by a later enrichment
   mutate {
-    id => "sdwan.app-mutate-logoriginal"
-    tag_on_failure => "_mutate_error-sdwan.app-logoriginal"
     add_field => {"[log][original]" => "%{[message]}"}
+  }
+  mutate {
     lowercase => ["[message]"]
   }
   
   grok {
     tag_on_failure => "_parsefailure_header"
-    match => { "message" => "(^(.*?)(<(?<pri>\d+)>)(\s)?(?<actual_msg>.*$))|(^(?<actual_msg>.*)$)" }
+    match => { "message" => "(^(.*?)(<(?<pri>\d+)>)(\s)?(?<[@metadata][tmp][actual_msg]>.*$))|(^(?<[@metadata][tmp][actual_msg]>.*)$)" }
```

Comment:
@nnovaes could you please proceed with "actual_msg" field instead of "[@metadata][tmp][actual_msg]", as we are trying to maintain single field name in all the processors - which would make things easier for us if any changes are to be made with that part of parsing.

Reply:
@Akhila-Y i moved the actual_msg into it's own field